domain-model/src/test/java/uk/gov/hmcts/cmc/domain/models/response/DefendantPaymentPlanValidationTest.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.cmc.domain.models.response;
+
+import org.junit.Test;
+import uk.gov.hmcts.cmc.domain.models.sampledata.response.SampleDefendantPaymentPlan;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.cmc.domain.utils.BeanValidator.validate;
+
+public class DefendantPaymentPlanValidationTest {
+    @Test
+    public void passesForValidSample() {
```

Comment:
What about testing scenario where payment plan is invalid? I could remove all validation annotations and tests would still pass.

Shall we change tests name to stick to `shouldXxx` format for consistency? I do not feel strong about it as current name ticks the box of having descriptive test names.